repo-blacklist.json

```diff
@@ -0,0 +1,78 @@
+[
+  "GTNH-Soartex-Overrides",
+  "FernFlower",
+  "DeveloperCapes",
+  "ForestryAPI",
+  "FTB-Library",
+  "D3Log",
+  "TravellersGear-Private",
+  "BON2",
+  "ImpactArchitectureModule",
+  "ForgePatcher",
+  "BQ3-Thermos-patch",
+  "Botdustries",
+  "TickProfiler",
+  "GTNH-O-Plenty",
+  "SpongePoweredMixin",
+  "QuestBook",
+  "GTNewHorizons.github.io",
+  "NewHorizonsServerCore",
+  "The-Aether-GTNH",
+  "MyEssentials-Core",
+  "NewHorizonsTweaker",
+  "SpongePoweredMixinUpstream",
+  "ASMHelper",
+  "BugTorch",
+  "RedstoneFlux-1.12-Legacy",
+  "FernFlowerLegacy",
+  "CoFHCore",
+  "RecEx",
+  "Dupes-Exploits-GTNH",
+  "CustomGTCapeHook-Cape-List",
+  "Forestry-Localization",
+  "Witchery",
+  "GTplusplus-OLD",
+  "Hbm-s-Nuclear-Tech-GIT",
+  "Obama",
+  "ZenScript",
+  "ProjectBlue",
+  "WarpTheory",
+  "Realistic-World-Gen",
+  "build-system-updater",
+  "MBPSPatcher",
+  "MyTown2-Client",
+  "GTNHCogs",
+  "TinkersGregworks",
+  "DreamAssemblerXXL",
+  "Alfheim",
+  "MicdoodleCore",
+  "AdvancedSystemsManager",
+  "HungerOverhaul",
+  "MyTown2",
+  "thaumcraft-api",
+  "MyPermissions",
+  "Opis",
```

Comment:
we should put this in the pack as it's one of the best tools to spot lag. SP players will have some in game way to audit their map